go/vt/servenv/grpc_server.go

```diff
@@ -64,6 +64,9 @@ var (
 	// GRPCCA is the CA to use if TLS is enabled
 	GRPCCA = flag.String("grpc_ca", "", "server CA to use for gRPC connections, requires TLS, and enforces client certificate check")
 
+	// GRPCServerCA if specified will combine server cert and server CA
+	GRPCServerCA = flag.String("grpc_server_ca", "", "combine server cert and server root CA, return full chain to client")
```

Comment:
What is expected to be provided as the value of this flag? the server root CA?
Can you add some text to the flag description to clarify this?

Reply:
thank you, updated phab, add more descriptive sentence